data/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
why did you add gitignore to data folder ?

Reply:
was autogenerated by android sudio, now it's deleted